pymc3/ode/utils.py

```diff
@@ -1,113 +1,127 @@
-import numpy as np
-import theano
-import theano.tensor as tt
-
-
-def make_sens_ic(n_states, n_theta, floatX):
-        """
-        The sensitivity matrix will always have consistent form. (n_states, n_states + n_theta)

```

Comment:
The 113 deletions it says are not actually deletions, probably an error in git. Because the lines are just deleted and then added back.